custom_components/csnet_home/api.py

```diff
@@ -158,7 +156,14 @@ async def async_get_elements_data(self):
             _LOGGER.error("Error during sensor data retrieval: %s", e)
             self.logged_in = False
             return None
-
+    
+    def get_current_temperature(self, element):
+        etype = element.get("elementType")
+        if etype == 5:
+            return element.get("settingTemperature") * 10
```

Comment:
Ah great. This means in some configuration the temperature "type" is changing?
What the elementType 5 is exactly?

We surely need to put all these information somewhere and write the Hitachi documentation...